aiger_analysis/test_count.py

```diff
@@ -9,6 +9,22 @@
 from aiger_analysis import count
 
 
+@given(aigh.Circuits, st.data())
+def test_bdd_transform(circ, data):
+    circ = circ.unroll(3)
```

Comment:
Why unroll? -> Separate unit test?

Reply:
In general circ will have a latch. I just unrolled to remove the latch. I could have also used `cutlatches`.

Reply:
Ah, seems more natural, to use cutlatches, but I don't mind.